webpack.config.js

```diff
@@ -22,15 +22,15 @@ const EXPLORER_APP = {
   envVars: {
     EXPLORER_APP_DOMAIN_REGEX_DEV: '^protocol-explorer\\.dev|^localhost:\\d{2,5}|^pr\\d+--gpui\\.review',
     EXPLORER_APP_DOMAIN_REGEX_STAGING: '^protocol-explorer\\.staging',
-    EXPLORER_APP_DOMAIN_REGEX_PROD: '^gnosis-protocol\\.io',
-    EXPLORER_APP_DOMAIN_REGEX_BARN: '^barn\\.gnosis-protocol\\.io',
+    EXPLORER_APP_DOMAIN_REGEX_PROD: '^explorer\\.cow\\.fi|^gnosis-protocol\\.io',
+    EXPLORER_APP_DOMAIN_REGEX_BARN: '^barn\\.explorer\\.cow\\.fi|^barn\\.gnosis-protocol\\.io',
 
     OPERATOR_URL_STAGING_MAINNET: 'https://protocol-mainnet.dev.gnosisdev.com/api',
     OPERATOR_URL_STAGING_RINKEBY: 'https://protocol-rinkeby.dev.gnosisdev.com/api',
     OPERATOR_URL_STAGING_XDAI: 'https://protocol-xdai.dev.gnosisdev.com/api',
-    OPERATOR_URL_PROD_MAINNET: 'https://protocol-mainnet.gnosis.io/api',
-    OPERATOR_URL_PROD_RINKEBY: 'https://protocol-rinkeby.gnosis.io/api',
-    OPERATOR_URL_PROD_XDAI: 'https://protocol-xdai.gnosis.io/api',
+    OPERATOR_URL_PROD_MAINNET: 'https://api.cow.fi/mainnet',
+    OPERATOR_URL_PROD_RINKEBY: 'https://api.cow.fi/rinkeby',
+    OPERATOR_URL_PROD_XDAI: 'https://api.cow.fi/xdai',
```

Comment:
Ah, another thing I failed to tell you.

We wanted to remove the suffix `/api` but we won't for now as it'll take longer on the backend side.
Please add it to them like it was with the previous domain:

```suggestion
    OPERATOR_URL_PROD_MAINNET: 'https://api.cow.fi/mainnet/api',
    OPERATOR_URL_PROD_RINKEBY: 'https://api.cow.fi/rinkeby/api',
    OPERATOR_URL_PROD_XDAI: 'https://api.cow.fi/xdai/api',
```